client/platform/desktop/frontend/components/Recent.vue

```diff
@@ -54,13 +57,34 @@ export default defineComponent({
       error, loading: checkingMedia, request, reset: resetError,
     } = useRequest();
 
-    async function open(dstype: DatasetType | 'text', directory = false) {
+    async function open(dstype: DatasetType | 'bulk' | 'text', directory = false) {
       const ret = await api.openFromDisk(dstype, directory);
-      if (!ret.canceled) {
-        pendingImportPayload.value = await request(() => api.importMedia(ret.filePaths[0]));
+      if (ret.canceled) {
+        return;
+      }
+
+      if (dstype === 'bulk') {
+        pendingImportPayload.value = await request(() => api.bulkImportMedia(ret.filePaths[0]));
```

Comment:
Down in the component you may want to handle when this returns an array of length 0.  Maybe it should be set to null to indicate that data didn't import.  To replicate point the bulk import at a folder that doesn't contain any valid datasets.